doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -605,6 +605,16 @@ Matter
 
 The issues in this section are related to the :ref:`ug_matter` protocol.
 
+.. rst-class:: v3-1-1 v3-1-0
+
+KRKNWK-20979: The Matter NUS service does not function correctly after the device is commissioned to the Matter network
+  After commissioning the device to the Matter network, the Matter Bluetooth Low Energy connection is dropped if a user attempts to connect to the device using the Matter NUS service.
+  This issue occurs because the SoftDevice Controller library automatically sets the :kconfig:option:`CONFIG_BT_LL_SOFTDEVICE_PERIPHERAL` Kconfig option to ``y``, which is not equipped to handle this scenario.
```

Comment:
Could add link to the SoftDevice Controller library?

Reply:
Actually, in all other places where the SoftDevice Controller library is mentioned in our docs, there are no links to it.